packages/components-js/tests/vrt/playwright/specs/states/pin-code.vrt.ts

```diff
@@ -0,0 +1,91 @@
+import { expect, type Page, test } from '@playwright/test';
+import {
+  baseSchemes,
+  baseThemes,
+  baseViewportWidth,
+  forceFocusHoverState,
+  forceFocusState,
+  forceHoverState,
+  getPlaygroundPseudoStatesMarkup,
+  type PrefersColorScheme,
+  setContentWithDesignSystem,
+} from '../../helpers';
+import { type Theme } from '@porsche-design-system/utilities-v2';
+
+const component = 'pin-code';
+
+const scenario = async (page: Page, theme: Theme, scheme?: PrefersColorScheme): Promise<void> => {
+  const markup = () => `<p-pin-code theme="${theme}" label="input gets hovered or focused"></p-pin-code>
+        <p-pin-code theme="${theme}" label="Disabled" disabled="true"></p-pin-code>
```

Comment:
Indentation for the entire block is off, e.g.
```suggestion
  <p-pin-code theme="${theme}" label="Disabled" disabled="true"></p-pin-code>
```